README.md

```diff
@@ -122,5 +130,5 @@ examples, extra testing, or new features please get in touch.
 [codeclimate-url]: https://codeclimate.com/github/senecajs/seneca-postgres-store
 [gitter-badge]: https://badges.gitter.im/Join%20Chat.svg
 [gitter-url]: https://gitter.im/senecajs/seneca
-[standard-store]: https://github.com/senecajs/seneca-standard-store
+[standard-query]: https://github.com/senecajs/seneca-standard-query
```

Comment:
There is another link at the top which you haven't changed.
Please be more careful next time. Thanks